modules/aws/agentless-gw/main.tf

```diff
@@ -42,19 +42,27 @@ module "gw_instance" {
   ami                               = var.ami
   instance_profile_name             = var.instance_profile_name
   additional_install_parameters     = var.additional_install_parameters
-  password                          = var.password
-  password_secret_name              = var.password_secret_name
+  admin_password                    = var.admin_password
+  secadmin_password                 = var.secadmin_password
+  sonarg_password                   = var.sonarg_password
+  sonargd_password                  = var.sonargd_password
+  admin_password_secret_name        = var.admin_password_secret_name
+  secadmin_password_secret_name     = var.secadmin_password_secret_name
+  sonarg_password_secret_name       = var.sonarg_password_secret_name
+  sonargd_password_secret_name      = var.sonargd_password_secret_name
   ssh_key_path                      = var.ssh_key_pair.ssh_private_key_file_path
   binaries_location                 = var.binaries_location
+  tarball_url                       = var.tarball_url
   hub_sonarw_public_key             = var.hub_sonarw_public_key
   hadr_dr_node                      = var.hadr_dr_node
   main_node_sonarw_public_key       = var.main_node_sonarw_public_key
   main_node_sonarw_private_key      = var.main_node_sonarw_private_key
-  proxy_info                        = var.ingress_communication_via_proxy
+  proxy_info                        = var.gw_proxy_info
```

Comment:
If you want to change this variable name, can you also change it at the Azure modules as well. We tried to create the same API and flow for AWS and Azure modules

Reply:
Do we want to change it? This was something Brian had changed, and I just copied over his code.

Reply:
The name of proxy_info sounds better to me. But I will let you decide if to change at all the places or revert this change

Reply:
Updated everything